pycose/keys/rsa.py

```diff
@@ -71,6 +74,50 @@ def from_dict(cls, cose_key: dict) -> 'RSAKey':
                    optional_params=_optional_params,
                    allow_unknown_key_attrs=True)
 
+    @classmethod
+    def from_cryptography_key(
+        cls,
+        ext_key: Union[rsa.RSAPrivateKey, rsa.RSAPublicKey],
+        optional_params: Optional[dict] = None
+    ) -> 'RSAKey':
+        """
+        Returns an initialized COSE Key object of type RSAKey.
+        :param ext_key: Python cryptography key.
+        :return: an initialized RSA key
+        """
+        if not cls._supports_cryptography_key_type(ext_key):
+            raise CoseIllegalKeyType(f"Unsupported key type: {type(ext_key)}")
+
+        if hasattr(ext_key, 'private_numbers'):
+            priv_nums = ext_key.private_numbers()
+            pub_nums = priv_nums.public_numbers
+        else:
+            priv_nums = None
+            pub_nums = ext_key.public_numbers()
+
+        cose_key = {}
+        if pub_nums:
+            cose_key.update({
+                RSAKpE: to_bstr(pub_nums.e),
+                RSAKpN: to_bstr(pub_nums.n),
+            })
```

Comment:
```suggestion
        cose_key = {
            RSAKpE: to_bstr(pub_nums.e),
            RSAKpN: to_bstr(pub_nums.n),
        }
```

Is `pub_nums` ever false-y? Seems like in either code path it gets assigned a value.